_posts/2023-10-11-october-bitdevs.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+type: socratic
+title: "BitDevs Amsterdam Conference Edition, October 11th 2023"
+meetup: https://www.meetup.com/bitdevs-amsterdam/
+published: true
+---
+
+## BitDevs Amsterdam meeting 007!
+
+It's time for a special edition of BitDevs, _The BitDevs Conference Edition!_ We hope to have a lot of friends from out of town joining, so we'll be meeting at a different place and different time than usual! Please read the details below carefully!
+
+### Wednesday, October 11th 2023 @ 3PM
+
+We will be meeting at the [MACA Amsterdam](https://www.maca.amsterdam/) in Amsterdam Noord:
+
+* Address: Ms. van Riemsdijkweg 61, 1033 RC Amsterdam
+* Google maps link: [https://maps.app.goo.gl/gZwCxM1EWxoAouU9A](https://maps.app.goo.gl/gZwCxM1EWxoAouU9A)
+
+<br />
+Doors open at 2:00PM and we will try to start promptly at 3PM. Light snacks will be provided and then we will join our friends for [De Bitcoin Meetup at Blast Galaxy](https://www.meetup.com/de-bitcoin-meetup/events/294584411/) for food and drinks (~10 min walk away, starting at 5PM).
```

Comment:
Suggestion: "Light snacks will be provided. We will joint our friends for .....".